genie_registry/maf.py

```diff
@@ -70,6 +70,8 @@ class maf(FileTypeFormat):
     _fileType = "maf"
 
     _process_kwargs = []
+    _allele_cols = ["REFERENCE_ALLELE", "TUMOR_SEQ_ALLELE1", "TUMOR_SEQ_ALLELE2"]
+    _allowed_alleles = ["A", "T", "C", "G", "N", " ", "-"]
```

Comment:
I would remove " "  and "-".

I like what you're thinking here for class attributes.  In the future, these classes should follow SOLID principles, right now it is doing too much.  I can imagine having a `Validate` class that `ValidateMaf` inherits from and then different compositions if necessary.

Reply:
Yes agreed, I originally had them defined in the `_validate` function and was debating between the two but was thinking future wise when we do want to start separating out into the separate classes we talked about (e.g Processing class, Validate class and regular read in data class), thought it would be easier if we put constants like these up in the class attributes given how big _validate is getting. Even before then, we might have other allele variable specific validation to add.